.gitignore

```diff
@@ -38,10 +38,6 @@ examples/*/django/
 examples/*/windows/
 examples/*/web/
 
-src/*/.tox
-src/*/.coverage
-src/*/.coverage.*
```

Comment:
These 3 patterns are already listed above without any directory, so they don't need to be listed per-directory as well.